src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -286,10 +312,11 @@ def ukn(s):
                 table.add_row((
                     s.name(),
                     ukn(s.nodename),
-                    ukn(s.container_id),
-                    ukn(s.version),
                     status,
-                    ukn(s.status_desc)))
+                    ukn(s.version),
+                    ukn(s.container_image_name),
+                    ukn(s.container_image)[0:12],
```

Comment:
Similar to https://github.com/ceph/ceph/pull/31828#discussion_r349878802 

Rename `container_image` -> `container_image_id` for clarity?